zh/pause-sync-of-tidbcluster.md

```diff
@@ -0,0 +1,123 @@
+---
+title: 暂停同步 Kubernetes 上的 TiDB 集群
+summary: 介绍如何暂停同步 Kubernetes 上的 TiDB 集群
+---
+
+# 暂停同步 Kubernetes 上的 TiDB 集群
+
+本文介绍如何通过配置文件暂停同步 Kubernetes 上的 TiDB 集群。
+
+## 什么是同步
+
+在TiDB Operator中，控制器运行一个非中止循环来调整 TiDB 集群状态。控制器会不断对比 TidbCluster 对象中记录的期望状态与 TiDB 集群的实际状态，并调整 Kubernetes 中的资源以驱动 TiDB 集群满足期望状态。这个不断调整的过程通常被称为 `同步`。更多细节参见[文档](architecture.md)。
+
+## 暂停同步的应用场景
+
+以下为一些可能用到暂停同步 Kubernetes 上 TiDB 集群的应用场景。
+
+1. 避免意外的滚动升级
+    假设你发现由于 TiDB Operator 新版本的兼容新问题或者操作失误，导致 TiDB 集群发生意外的滚动升级。你可以参照下面的方式来暂停 TiDB 集群的同步过程，避免 TiDB 集群被错误地升级。
+
+2. 维护时间窗口
+    在某些情况下，只允许在特定时间窗口内配置 TiDB 集群的状态。因此可以在维护时间窗口之外的时间段暂停 TiDB 集群的同步过程，这样在维护时间窗口之外对 TiDB 集群的任何配置都不会生效；在维护时间窗口时，可以通过恢复 TiDB 集群同步来允许配置 TiDB 集群的状态。
+
+## 暂停同步 TiDB 集群
+
+在 TiDB Cluster 配置文件的 `spec` 配置项中新增 `paused: true`，并应用该配置文件，TiDB 集群各组件 (PD、TiKV、TiDB、Pump) 的同步过程将会被暂停。
+
+1. 修改配置文件如下：
+
+    {{< copyable "shell-regular" >}}
```

Comment:
```suggestion
    {{< copyable "" >}}
```

Reply:
This will add a `$` before the code block. e.g.:
![image](https://user-images.githubusercontent.com/59654584/101612703-e68bad00-3a45-11eb-9307-8a4d57b77c60.png)